### PR TITLE
Add omero-web 5.14.0 upgrade notes for sys-admins

### PR DESCRIPTION
**Pull request by will-moore**
cc @jburel

### CODE REVIEW

File: MIGRATION_TO_DJANGO_32_GUIDE.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
 
 ## Introduction
 
-Django 3.2.x is now the major version that OMERO.web depends upon. All the way back to 5.6.0, this was previously Django 1.11.x. This Django version conservatism has helped us establish a very fertile, stable environment for plugin developers. However, while the usage of the plethora of Django functionality available in core OMERO.web is quite limited, upgrading your plugin to support this new version requires careful consideration.
+Django 3.2.x is now the major version that OMERO.web depends upon. For sys-admins, the installation of `omero-web==5.14.0` will also upgrade Django and Django-cors-headers as required.
```
Comment (jburel) [inline MIGRATION_TO_DJANGO_32_GUIDE.md:5]: strange spelling of sysadmin. Maybe preferable to use ``system administrators``

File: MIGRATION_TO_DJANGO_32_GUIDE.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
...
+You will also need to upgrade your `omero-web` plugin apps to versions compatible with Django 3.2.
+OME has recently updated `omero-iviewer`, `omero-figure`, `omero-parade`, `omero-mapr` among others,
```
Comment (jburel) [inline MIGRATION_TO_DJANGO_32_GUIDE.md:7]: I see what you want to distinguish who is supporting which app but unless it is indicated I don't think people will release that we are not  the maintainers ``webtagging``.
Maybe preferable to go ``popular apps like `omero-iviewer` etc have been updated.``

File: MIGRATION_TO_DJANGO_32_GUIDE.md [hunk 3]
```diff
@@ -2,7 +2,16 @@
...
+and `omero-webtagging` has also been updated.
+
+You many need to uninstall other Django apps such as https://pypi.org/project/django-cookies-samesite/ which is not needed for `Django 3.1+`.
```
Comment (jburel) [inline MIGRATION_TO_DJANGO_32_GUIDE.md:10]: Do you know of others maybe add an official link from Django

Comment (will-moore) [reply MIGRATION_TO_DJANGO_32_GUIDE.md:10]: I don't know of others and I don't think Django itself has any such listing of 3rd-party apps.

Comment (jburel) [reply MIGRATION_TO_DJANGO_32_GUIDE.md:10]: There is https://djangopackages.org/categories/apps/ 
but I am not convinced it is useful here

Comment (will-moore) [reply MIGRATION_TO_DJANGO_32_GUIDE.md:10]: No, I don't think that adds anything here.

**Comment by jburel** [comment]
Thanks @will-moore

**Pull request merged**